bring/src/main/java/com/bobocode/petros/bring/utils/ScanningUtils.java

```diff
@@ -12,10 +12,7 @@
 import org.reflections.util.ConfigurationBuilder;
 
 import java.lang.reflect.Type;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Objects;
-import java.util.Set;
+import java.util.*;
```

Comment:
MINOR. Using wildcard in imports is a bad practice

Reply:
It is possible to enable jacoco rule to prohibit all wildcard imports, isn't it?
Is it possible to configure IDEA to import only required classes? Have you done it on your current/previous projects?